src/cli/commands/init.js

```diff
@@ -89,12 +91,17 @@ export async function run(config: Config, reporter: Reporter, flags: Object, arg
       question: 'license',
       default: String(config.getOption('init-license')),
     },
+    {
+      key: 'private',
+      question: 'private',
+      default: '',
```

Comment:
This is a valid point. I was thinking yn might error if the user types in something like "i guess so" as the answer, but I suspect it will just try to convert it anyway. As long as yn doesn't throw an exception, then I would be OK with no validation.